### PR TITLE
[FR] Update `group-info`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the meta page `group-info`.

Context: #5918 and #5921

### CODE REVIEW

File: meta/group-info/fr.yaml [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
     dev: osu!dev
-    media: créateur de osu!media
-    support: équipe de soutien
-    tournaments: gestionnaire des tournois
+    media: Créateur de osu!media
+    support: Support Team
+    tournaments: Gestion des tournois
 gmt:
-  all_mods: tous les modérateurs
+  all_mods: Tout les modérateurs
```
Comment (Ganondorf-1) [inline meta/group-info/fr.yaml:12]:
```suggestion
  all_mods: Tous les modérateurs
```

File: meta/group-info/fr.yaml [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
-    evaluation: évaluation
-    moderation: modération
-    structural: structurelle
+    communication: Communication
+    evaluation: Évaluation
+    moderation: Modération
+    structural: Structurel
 languages:
-  partial: une certaine <language>
+  partial: un peut de <language>
```
Comment (Ganondorf-1) [inline meta/group-info/fr.yaml:31]:
```suggestion
  partial: un peu de <language>
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: meta/group-info/fr.yaml [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
-    chat: modérateur de tchat
+    academy: Créateur de la osu!academy
+    admin: Administration
+    chat: Modérateur de tchat
     dev: osu!dev
-    media: créateur de osu!media
-    support: équipe de soutien
-    tournaments: gestionnaire des tournois
+    media: Créateur de osu!media
+    support: Support Team
```
Comment (Ganondorf-1) [inline meta/group-info/fr.yaml:9]:
```suggestion
    support: Équipe du support
```

File: meta/group-info/fr.yaml [hunk 4]
```diff
@@ -1,34 +1,34 @@
...
 nat:
   areas:
-    communication: communication
-    evaluation: évaluation
-    moderation: modération
-    structural: structurelle
+    communication: Communication
+    evaluation: Évaluation
+    moderation: Modération
+    structural: Structurel
```
Comment (Ganondorf-1) [inline meta/group-info/fr.yaml:29]:
```suggestion
    structural: Structure
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — DISMISSED** [review]

**Review by TPGPL — COMMENTED** [review]
is there a reason the phrases were capitalised?

**Comment by DoryanR** [comment]
No not really, I'll put them in lower case

File: meta/group-info/fr.yaml [hunk 5]
```diff
@@ -1,34 +1,34 @@
 separator: ", "
 alumni:
   roles:
-    academy: fondateur de la osu!academy
-    admin: administration
-    chat: modérateur de tchat
+    academy: Créateur de la osu!academy
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:4]:
```suggestion
    academy: créateur de la osu!academy
```

File: meta/group-info/fr.yaml [hunk 6]
```diff
@@ -1,34 +1,34 @@
...
+    admin: Administration
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:5]:
```suggestion
    admin: administration
```

File: meta/group-info/fr.yaml [hunk 7]
```diff
@@ -1,34 +1,34 @@
...
+    chat: Modérateur de tchat
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:6]:
```suggestion
    chat: modérateur de tchat
```

File: meta/group-info/fr.yaml [hunk 8]
```diff
@@ -1,34 +1,34 @@
...
     dev: osu!dev
-    media: créateur de osu!media
-    support: équipe de soutien
-    tournaments: gestionnaire des tournois
+    media: Créateur de osu!media
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:8]:
```suggestion
    media: créateur de osu!media
```

File: meta/group-info/fr.yaml [hunk 9]
```diff
@@ -1,34 +1,34 @@
...
+    support: Équipe du support
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:9]:
```suggestion
    support: équipe du support
```

File: meta/group-info/fr.yaml [hunk 10]
```diff
@@ -1,34 +1,34 @@
...
+    tournaments: Gestion des tournois
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:10]:
```suggestion
    tournaments: gestion des tournois
```

File: meta/group-info/fr.yaml [hunk 11]
```diff
@@ -1,34 +1,34 @@
...
 gmt:
-  all_mods: tous les modérateurs
+  all_mods: Tous les modérateurs
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:12]:
```suggestion
  all_mods: tous les modérateurs
```

File: meta/group-info/fr.yaml [hunk 12]
```diff
@@ -1,34 +1,34 @@
...
-    chat: modérateur du tchat
-    forum: modérateur du forum
-    team: équipe de présence officiel
-    dev: osu! development
-    player: support au joueur
-    support: support technique
-    tournaments: gestionnaire des tournois
-    wiki: administration du wiki
-    mapping: modération de la communauté modding/mapping
+    beatmap: Modération des beatmaps
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:14]:
```suggestion
    beatmap: modération des beatmaps
```

File: meta/group-info/fr.yaml [hunk 13]
```diff
@@ -1,34 +1,34 @@
...
+    chat: Modération du tchat
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:15]:
```suggestion
    chat: modération du tchat
```

File: meta/group-info/fr.yaml [hunk 14]
```diff
@@ -1,34 +1,34 @@
...
+    forum: Modération du forum
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:16]:
```suggestion
    forum: modération du forum
```

File: meta/group-info/fr.yaml [hunk 15]
```diff
@@ -1,34 +1,34 @@
...
+    team: Présence officielle de l'équipe
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:17]:
```suggestion
    team: présence officielle de l'équipe
```

File: meta/group-info/fr.yaml [hunk 16]
```diff
@@ -1,34 +1,34 @@
...
+    dev: Développement d'osu!
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:18]:
```suggestion
    dev: développement d'osu!
```

File: meta/group-info/fr.yaml [hunk 17]
```diff
@@ -1,34 +1,34 @@
...
+    player: Support aux joueurs
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:19]:
```suggestion
    player: support aux joueurs
```

File: meta/group-info/fr.yaml [hunk 18]
```diff
@@ -1,34 +1,34 @@
...
+    support: Support technique
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:20]:
```suggestion
    support: support technique
```

File: meta/group-info/fr.yaml [hunk 19]
```diff
@@ -1,34 +1,34 @@
...
+    tournaments: Gestion des tournois
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:21]:
```suggestion
    tournaments: gestion des tournois
```

File: meta/group-info/fr.yaml [hunk 20]
```diff
@@ -1,34 +1,34 @@
...
+    wiki: Administration du wiki
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:22]:
```suggestion
    wiki: administration du wiki
```

File: meta/group-info/fr.yaml [hunk 21]
```diff
@@ -1,34 +1,34 @@
...
+    mapping: Modération de la communauté mapping/modding
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:23]:
```suggestion
    mapping: modération de la communauté modding/mapping
```

File: meta/group-info/fr.yaml [hunk 22]
```diff
@@ -1,34 +1,34 @@
...
 nat:
   areas:
-    communication: communication
-    evaluation: évaluation
-    moderation: modération
-    structural: structurelle
+    communication: Communication
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:26]:
```suggestion
    communication: communication
```

File: meta/group-info/fr.yaml [hunk 23]
```diff
@@ -1,34 +1,34 @@
...
+    evaluation: Évaluation
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:27]:
```suggestion
    evaluation: évaluation
```

File: meta/group-info/fr.yaml [hunk 24]
```diff
@@ -1,34 +1,34 @@
...
+    moderation: Modération
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:28]:
```suggestion
    moderation: modération
```

File: meta/group-info/fr.yaml [hunk 25]
```diff
@@ -1,34 +1,34 @@
...
+    structural: Structure
```
Comment (DoryanR) [inline meta/group-info/fr.yaml:29]:
```suggestion
    structural: structure
```

**Review by TPGPL — APPROVED** [review]

**Pull request merged**